parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -164,7 +135,7 @@ protected LogicalTypeAnnotation fromString(List<String> params) {
    */
   public abstract void accept(LogicalTypeAnnotationVisitor logicalTypeAnnotationVisitor);
 
-  public abstract LogicalTypes getType();
+  protected abstract LogicalTypeToken getType();
```

Comment:
nit: package private would be fine here.

Reply:
Good point. I would say if something is not necessary to be public then restrict its access before committing. The later the harder to remove from the public API.